packages/ui/src/components/Designer/Sidebar/DetailView/BarCodePropEditor.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from 'react';
+import { SchemaForUI, DEFAULT_FONT_COLOR } from '@pdfme/common';
+import { I18nContext } from '../../../../contexts';
+import { SidebarProps } from '..';
+import { XMarkIcon } from '@heroicons/react/24/outline';
+
+const inputStyle = {
+  width: '90%',
+  color: '#333',
+  background: 'none',
+  borderRadius: 2,
+  border: '1px solid #767676',
+};
+
+const ColorInputSet = (props: {
```

Comment:
I see that there's already a ColorInputSet, which is at least very similar to what you're doing here, in the `TextPropEditor`. 

I think we should consolidate this code and make it shared.

Reply:
actually, whilst this should be done, it opens some design decisions that can happen separately. 
We can comment about this on the upstream PR 👍

Reply:
Ah yes, this did cross my mind, the src directory could do with a bit of a clean-up and start making prop editor components shared across the UI. 👍